akka-docs/rst/scala/code/docs/persistence/PersistentActorExample.scala

```diff
@@ -29,14 +29,15 @@ class ExamplePersistentActor extends PersistentActor {
     case SnapshotOffer(_, snapshot: ExampleState) => state = snapshot
   }
 
+  val snapShotInterval = 1000
   val receiveCommand: Receive = {
     case Cmd(data) =>
-      persist(Evt(s"${data}-${numEvents}"))(updateState)
-      persist(Evt(s"${data}-${numEvents + 1}")) { event =>
```

Comment:
I reduced the number of `persist()` calls from two to one.

The associated text section mentions multiple `persist()` calls, but I think we don't need the multiple `persist()` calls in this very first example on the page, 

http://doc.akka.io/docs/akka/2.4/scala/persistence.html#Event_sourcing

<img width="712" alt="2017-03-02 0 04 38" src="https://cloud.githubusercontent.com/assets/7414320/23465530/deee83fa-fedb-11e6-8be5-97ee62da78cf.png">

Because multiple calls are shown in a later example on the same page.

http://doc.akka.io/docs/akka/2.4/scala/persistence.html#Relaxed_local_consistency_requirements_and_high_throughput_use-cases
<img width="402" alt="2017-03-02 0 05 38" src="https://cloud.githubusercontent.com/assets/7414320/23465578/02d35098-fedc-11e6-9b83-791cfbb2d813.png">